src/Firebase.Notifications/Android/Impl.uno

```diff
@@ -53,10 +53,13 @@ namespace Firebase.Notifications
                 new com.fuse.Activity.IntentListener() {
                     public void onIntent (Intent newIntent) {
                         String jsonStr = com.fuse.firebase.Notifications.PushNotificationReceiver.ToJsonString(newIntent);
-                        @{OnRecieve(string,bool):Call(jsonStr, false)};
+                        if( jsonStr != "{}")
+                        {
+                            @{OnRecieve(string,bool):Call(jsonStr, false)};
+                        }
```

Comment:
I really didn't what you need to know really, but this code snippet handle the following case :
To distignuesh between Normal app open and Open from notification tray, because of we are listening `android.intent.action.MAIN` due to Android 8+ app start new policy.
The `android.intent.action.MAIN` filter catch each app start, while the `getIntent()` on `java` provides me the data that attached with app launch. Normal open holds nothing while the open from Notification tray holds the notification data ;)
Hope this goes clear

Reply:
Thanks for explaining. If I understand correctly, you're saying that this prevents us from calling OnReceive (with empty data) when the app was started normally? If so, I think it would be nice to factor that out into a separate commit that explains that (in the commit message), so future generations will understand what's going on.

I'm not feeling very strongly about this, though. If it's a huge hassle, we can just leave it. Perhaps add a comment? Dunno...